datatypes/src/operations/reproject.rs

```diff
@@ -474,10 +476,45 @@ pub fn reproject_query<S: AxisAlignedRectangle>(
     })
 }
 
+/// Reproject a bounding box to the `target` projection and return the input and output bounding box
+/// as a pair where both elements cover the same area of the original `source_bbox` in WGS84.
+pub fn reproject_and_unify_bbox<T: AxisAlignedRectangle>(
```

Comment:
let's keep the other variant. this is way to complex.

Reply:
4cc7328bae331ccfa33462032ed881ca5f52c09f